src/Grid/Clearfix.js

```diff
@@ -0,0 +1,2 @@
+import { Clearfix } from 'react-bootstrap'
```

Comment:
cant you just
```js
export { Clearfix } from 'react-bootstrap'
```

Reply:
@ohadlevy I wanted to be consistent with other files like:
https://github.com/patternfly/patternfly-react/blob/master/src/MenuItem/MenuItem.js

Exporting components from `react-bootstrap` is a common task, i think it should be consistent everywhere.

Reply:
@ohadlevy @sharvit of course i'm OK with making the two line files one line files. Let's just be consistent...

Feel free to update the others alongside this PR if you can. Otherwise can be do a subsequent PR.